scripts/ovl.py

```diff
@@ -1238,8 +1238,11 @@ def _push_single_target(src, dst):
           #   dest_dir/A
           dst_root = root if dst_exists else root[len(src):].lstrip('/')
           for name in files:
+            # trailing ../ for the src_dir name
+            relative_path = os.path.relpath(dst_root, f'{src}/../')
+            dst_filepath = f'{dst}/{name}' if relative_path == '.' else f'{dst}/{relative_path}/{name}'
```

Comment:
and use `os.path.join` instead.

Reply:
if `dst` path does not exist in the client's drive, the result of `os.path.join(dst, src)` is still src path

for example:
```python
src = '/home/kellly/src_dir'
dst = '/home/myuser/workspace'
os.path.join(dst, src) # output '/home/kellly/src_dir'
```

so I join the path by strcat.